src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,90 @@
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+
+import { TableRow } from '../TableRow/TableRow';
+import { Loader } from '../Loader';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loader, setLoader] = useState(false);
```

Comment:
let's use `is/has` for boolean type, also in this case you can set `isLoading` as `true` by default